collector/collector.go

```diff
@@ -57,7 +62,11 @@ type Storage interface {
 
 // Run runs the collector on the Log specified in cfg, and stores the collected
 // data in st.  Run doesn't return unless an error occurs or ctx expires.
-func Run(ctx context.Context, cfg *Config, cl *http.Client, st Storage) error {
+//
+// If cfg.AddChainPeriod is set to 0, i.e. the collector will not be issuing and
+// submitting certificates to the Log, then set ca to nil, as it will not be
+// used.
```

Comment:
Moving `ca` to `Config` would probably allow doing without this comment, I'd say.

Reply:
Yup, done.